Addon.Episever.EnvironmentSynchronizer.Test/Synchronizers/SiteDefinitions/SiteDefinitionSynchronizerTests.cs

```diff
@@ -0,0 +1,271 @@
+﻿using Addon.Episerver.EnvironmentSynchronizer.Configuration;
+using Addon.Episerver.EnvironmentSynchronizer.Synchronizers.SiteDefinitions;
+using EPiServer.Core;
+using EPiServer.DataAbstraction;
+using EPiServer.Security;
+using EPiServer.Web;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Addon.Episever.EnvironmentSynchronizer.Test.Synchronizers.SiteDefinitions
+{
+	public class SiteDefinitionSynchronizerTests
```

Comment:
Suggestion for easier readability:
1. Rename this test class to RemoveAccessForEveryoneRoleTests. This means that this class is focused on testing that method and you don't need to include that in every test name.
2. Rename the tests so that are written in an easy to read format. You should be able to give the list of names to a project manager, and they should be able to understand. Example:

RemoveAccessForEveryoneRole_NullStartPage_Nothing -> When_everyone_is_not_part_of_accessrights_no_changes_should_be_done